src/Microsoft.IdentityModel.Protocols.SignedHttpRequest/SignedHttpRequestUtilities.cs

```diff
@@ -116,10 +116,12 @@ internal static async Task<JsonWebKey> DecryptSymmetricPopKeyAsync(JsonWebTokenH
             var tokenDecryptionParameters = new TokenValidationParameters()
             {
                 TokenDecryptionKeys = decryptionKeys,
+#pragma warning disable CS0618 // TODO: If this is a valid use of RequireSignedTokens, perhaps this property should be left as not obsolete to not disrupt our consumers.
```

Comment:
Some legacy tokens are/can be unsigned.

Reply:
Makes sense, I'll update to no longer mark this property as obsolete then.